src/core/namespace.c

```diff
@@ -724,6 +752,20 @@ static int mount_empty_dir(const MountEntry *m) {
         return 1;
 }
 
+static int mount_tmpfs(const MountEntry *m) {
+        assert(m);
+
+        /* First, get rid of everything that is below if there is anything. Then, overmount with our new empty dir */
```

Comment:
Comment is a copy-paste from mount_empty_dir().  Should maybe be changed to "our new tmpfs".

Reply:
Thanks. updated.